packages/flutter/lib/src/widgets/interactive_viewer.dart

```diff
@@ -145,8 +144,145 @@ class InteractiveViewer extends StatefulWidget {
                  boundaryMargin.left.isFinite),
        ),
        constrained = false,
+       _transformChild = true,
        child = null;
 
+  /// Using [InteractiveViewer.raw] will not transform the child based on the [Matrix4]
```

Comment:
(re-adding the comment since the parent comment was resolved before I submitted this).
First sentence for a briefly description of what this does. You can expand on it later.

Maybe: Creates an [InteractiveViewer] that does not apply its current transform to its child widget.